docs/content/docs/1.getting-started/7.ai/1.mcp.md

```
@@ -28,33 +28,47 @@ The Nuxt UI MCP server provides the following tools organized by category:
 ### Migration Tools
 
-- **`get_migration_guide`**: Retrieves version-specific migration guides and upgrade instructions
+- **`get-migration-guide`**: Retrieves version-specific migration guides and upgrade instructions
+
+## Limit available tools
+
+Set the optional `X-MCP-Tools` HTTP header to a comma-separated list of tool names to expose only the tools your assistant needs. This helps to reduce the context defined in your assistant.
+
+Use the exact kebab-case MCP tool names, for example:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)